### DISCUSSION TITLE
specified as URL parameters

### DISCUSSION THREAD

**Discussion by mayost** [Xpra-org/xpra-html5#293] [Q&A]
Hello
FEDORA 39 server
XPRA 5.06

When I connect through the client with specifying parameters in the URL
It seems it is not sending the parameters
For example:
http://192.168.10.91:10000/?keyboard_layout=fr

When I run the command there:
setxkbmap -query
I see that the LAYOUT still remains on US

Thanks

**Comment by totaam** [comment]
Duplicate of https://github.com/Xpra-org/xpra-html5/pull/292
